src/components/ChecModal.vue

```diff
@@ -73,12 +88,15 @@ export default {
 .modal {
   &__overlay {
     @apply fixed top-0 left-0 w-full h-full z-50 flex justify-center items-center overflow-scroll;
-    // Special / overlay
```

Comment:
We had this comment here deliberately, because otherwise we'd have expected to use a colour from a variable or Tailwind mixiny thing - could you put the comment back, and apply it to the new dark colour you added on line 94 please?